tests/transforms/test_binarize_extract_bridge.py

```diff
@@ -0,0 +1,72 @@
+import json
+import pytest
+
+from ophys_etl.types import DenseROI
+from ophys_etl.transforms.binarize_extract_bridge import \
+        BinarizeToExtractBridge
+
+
+@pytest.fixture
+def mock_movie_path(tmp_path):
```

Comment:
This is fine, you could also use the fixture in:

https://github.com/AllenInstitute/ophys_etl_pipelines/blob/master/tests/conftest.py

Reply:
thought about it, but didn't need it. I like it small and self-contained if possible